authorizenet.gemspec

```diff
@@ -11,9 +11,10 @@ Gem::Specification.new do |s|
   s.homepage = "https://github.com/AuthorizeNet/sdk-ruby"
   s.license = "https://github.com/AuthorizeNet/sdk-ruby/blob/master/license.txt"
 
-  s.required_ruby_version     = '>= 1.9.3'
+  s.required_ruby_version     = '>= 2.2.5'
   s.required_rubygems_version = '>= 1.3.6'
 
+  s.add_runtime_dependency 'activesupport', '= 4.2.6'
```

Comment:
This prevents using the gem with Rails 5. We can't upgrade to 1.9.x now. Is there a reason to not lock to `'>= 4.2.6'`?